linchpin/provision/roles/common/tasks/main.yml

```diff
@@ -13,3 +13,7 @@
   set_fact:
     topology_outputs: {}
 
+- name: "assign debug_mode value"
+  set_fact:
+    debug_mode: "{{ _debug_mode }}"
```

Comment:
This is the reason for `debug_mode` and `_debug_mode`.  This is the same as `async` and `_async` on line 2

Reply:
Oh, I see. You were just trying to confuse me by being clever. :)

Reply:
@14rcole I wonder if we need to set the debug mode this way. If `_debug_mode` is what is used in the evars section of linchpin.constants/linchpin.conf, maybe we should be consistent in the code throughout? It's kind of unnecessary. It is in code, however, so if you don't want to change it, it's probably fine.

Reply:
I don't mind changing it, but should the `_async` option be changed as well?  Also, is there any point in the underscore if the variable isn't being renamed here?

Reply:
The reason the `_async` has an underscore is because ansible already has `async` used and would conflict. LinchPin uses `async: "{{ _async }}" in plays. I am actually fine with taking out the underscore and just having `debug_mode` since you didn't use `_debug` which may have conflicted with something in ansible.

Reply:
I see.  I'll change that